cmake/expat/expat_config.h.cmake

```diff
@@ -0,0 +1,91 @@
+/* expat_config.h.in.  Generated from configure.in by autoheader.  */
```

Comment:
Why do you need this file at all, when it wasn't previously needed? This doesn't seem right, surely this won't be accurate for all possible compilers.